compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/Actuals.jsWasm.kt

```diff
@@ -16,4 +16,9 @@
 
 package androidx.compose.ui
 
+internal actual fun currentTimeMillis(): Long {
+    // TODO https://youtrack.jetbrains.com/issue/CMP-7152/Implement-currentTimeMillis-for-web
```

Comment:
Should it be fixed before merging this PR, or later in jb-main? 

JFYI: we have `currentNanoTime()` from skiko.

Reply:
If it's not blocking CI - later